src/main/java/stream/flarebot/flarebot/FlareBot.java

```diff
@@ -222,6 +199,9 @@ public static void main(String[] args) throws Exception {
         String tkn = config.getString("bot.token").get();
 
         new CassandraController().init(config);
+
+        /*if (config.getString("bot.pasteAccessKey").isPresent())
```

Comment:
why is this commented out?

Reply:
All of that is commented out to make use of a global config variable instead. Much cleaner and generally more efficient code.